src/Microsoft.Diagnostics.EventFlow.Inputs.MicrosoftLogging/Logger/EventFlowLogger.cs

```diff
@@ -113,16 +116,12 @@ private LogLevel ToLogLevel(Extensions.Logging.LogLevel loggerLevel)
             return LogLevel.Verbose;
         }
 
-        private void InvokeAndReport(Func<DictionaryExtenstions.AddResult> action)
+        private void AddPayloadProperty(IDictionary<string, object> payload, string key, object value)
```

Comment:
you probably can remove this method, since it just proxies PayloadDictionaryUtilities.AddPayloadProperty